step-strikers-app/step-strikers-app/Classes/Armor.swift

```diff
@@ -0,0 +1,90 @@
+//
+//  Armor.swift
+//  step-strikers-app
+//
+//  Created by Jalyn Derry on 2/9/23.
+//
+
+
+protocol Armor {
```

Comment:
Are we having condition for armor?

Reply:
what would work better in firebase, enumerated types or ints?

Reply:
Ints would be great!